src/services/types.ts

```diff
@@ -542,6 +542,13 @@ namespace ts {
     export interface DocumentSpan {
         textSpan: TextSpan;
         fileName: string;
+
+        /**
+         * If the span represents a location that was remapped (e.g. via a .d.ts.map file),
+         * then the original filename and span will be specified here

```

Comment:
Is it worth noting that it may have been mapped multiple times and that (from what I can tell) this isnt currently the original-original, but rather just one step back from the final mapping?

Reply:
I'll update the code to return the true original